app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,40 @@
+class FriendshipsController < ApplicationController
+  def create
+    return if current_user.id == params[:user_id]
+    return unless current_user.viable_friend?(User.find(params[:user_id]))
+
+    # @friendship = current_user.send_a_request(params[:user_id])
```

Comment:
**OPTIONAL**
* Comments: It’s better to delete commented piece of codes. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from Pathwright).